README.md

```diff
@@ -112,6 +115,40 @@ The CI automatically builds the lib, creates a release named after the tag and i
 
 This package URL can also be used to reference the dependency for NPM in package.json, see below
 
+### Automatic tag in CI
+
+On merge of a PR on main branch the CI will create an automatic tag of type
+<branch_name>\_CI\_<short_commit>
```

Comment:
So the difference between manual "dev" tags created on a branch and tags automatically created on main is the "TAG" vs "CI" part if I see right. Just wondering if other terms would be more explicit to identify the difference without becoming too long. WDYT of "DEV_TAG" instead of "TAG"?

Reply:
just _DEV_ instead of _TAG_ ??